prdoc/pr_7169.prdoc

```diff
@@ -0,0 +1,14 @@
+title: 'xcm: fix DenyThenTry when work with multiple Deny tuples'
+doc:
+- audience: Runtime User
+  description: |-
+    This PR changes the behavior of DenyThenTry to fix #7148
+    If any of the tuple elements returns `Err(())`, the execution stops. 
+    Else, `Ok(_)` is returned if all elements accept the message.
+crates:
+- name: staging-xcm-executor
+  bump: patch
+- name: staging-xcm-builder
+  bump: patch
+- name: bridge-hub-common
+  bump: patch
```

Comment:
```suggestion
- name: staging-xcm-executor
  bump: minor
- name: staging-xcm-builder
  bump: minor
- name: bridge-hub-common
  bump: minor
```

Should all be minor since public API was modified